jolt-core/src/jolt/vm/bytecode/read_checking.rs

```diff
@@ -365,60 +412,61 @@ impl<F: JoltField> BatchableSumcheckInstance<F> for ReadCheckingSumcheck<F> {
         );
     }
 
-    fn expected_output_claim(&self, r: &[F]) -> F {
-        let ra_claim = self.ra_claim.as_ref().expect("ra_claim not set");
+    fn expected_output_claim(
+        &self,
+        accumulator: Option<Rc<RefCell<VerifierOpeningAccumulator<F>>>>,
+        r: &[F],
+    ) -> F {
+        let (_, ra_claim) = accumulator
+            .as_ref()
+            .unwrap()
+            .borrow()
+            .get_committed_polynomial_opening(CommittedPolynomial::BytecodeRa, self.sumcheck_id);
+
         let r: Vec<_> = r.iter().rev().copied().collect();
 
         // Verify sumcheck_claim = ra_claim * val_eval
-        *ra_claim * self.val_poly.evaluate(&r)
+        ra_claim * self.val_poly.evaluate(&r)
+    }
+
+    fn normalize_opening_point(&self, opening_point: &[F]) -> OpeningPoint<BIG_ENDIAN, F> {
+        OpeningPoint::new(opening_point.iter().copied().rev().collect())
```

Comment:
Should only the r_cycle be reversed since we are binding it LowToHigh, but r_address is HighToLow?

Reply:
Nvm, it's LowToHigh in both if d=1